fixtures/sample.json

```diff
@@ -89,7 +89,7 @@
             "description": "The amount of memory available on the client machine, measured in MB.",
             "name": "Memory",
             "source_name": "memory_avg",
-            "tooltip": "{y}% of all records have {x} MB of memory",
+            "tooltip": "{y}% of {pop} records have {x} MB of memory",
```

Comment:
Good call. We'll need to update our models via the admin when this is merged and pushed so they include the population in the hover strings.